src/traits/widget.rs

```diff
@@ -360,6 +360,13 @@ pub trait WidgetTrait: ::FFIWidget + ::GObjectTrait {
         unsafe { to_bool(ffi::gtk_widget_get_double_buffered(self.unwrap_widget())) }
     }
 
+    fn get_window(&self) -> Option<gdk::Window> {
+        unsafe {
+            let tmp = ffi::gtk_widget_get_window(self.unwrap_widget());
+            from_glib_none(tmp)
```

Comment:
The temporary variable isn't of much use now so let's remove it here and in the other PR to keep the style consistent with other crates.